pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
> So, what is the problem with proposed fix at line 203 ?

Line 203 switched ```#ifdef BIT64``` to ```#ifndef BIT64```. The problem is in cmake we are defining BIT64 [here](https://github.com/Microsoft/ChakraCore/blob/linux/CMakeLists.txt). This change thus switches cmake build code path, changing our __int64 def on 64bit unix from ```long``` to ```long long```.

> Second;
> The problem is when __int64 is long, _Is_integral has duplicate definitions. See https://github.com/Microsoft/ChakraCore/pull/157/files#diff-050fe98eebc72bf7b89245109bb89320L311 and line 317

Please define ```PLATFORM_UNIX``` on APPLE. Existing line 309 avoids the duplicated definitions problem.

Without ```BIT64``` and ```PLATFORM_UNIX``` defs, you could run into more problems down the line.